Documentation/Blazorise.Docs/Models/Snippets.generated.cs

```diff
@@ -8105,7 +8118,44 @@ protected override async Task OnInitializedAsync()
 
 @code {
     private DataGrid<Employee> dataGrid;
-    private List<Employee> employeeList = new() { new() { FirstName = ""David"", LastName = ""Moreira"", Gender = ""M"" }, new() { FirstName = ""MLaden"", LastName = ""Macanovic"", Gender = ""M"" }, new() { FirstName = ""John"", LastName = ""Doe"", Gender = ""M"" }, new() { FirstName = ""Ana"", LastName = ""Chamberlain"", Gender = ""F"" }, new() { FirstName = ""Jessica"", LastName = ""Winston"", Gender = ""F"" } };
+    private List<Employee> employeeList = new()
```

Comment:
You've removed the Children assignment I had added:
![image](https://github.com/user-attachments/assets/4f54b65f-91d3-4973-a1f3-824a64effe4e)

Also why change these names, and not the ones from the other example?

Reply:
I missed that, sorry.

For the new names. Just for the change. And it feels strange to have my name still there 😅

Reply:
The example above also has your name...
I'd go through the docs and change everywhere if that's a concern.

Reply:
> I'd go through the docs and change everywhere if that's a concern.

That's the next step.